aten/src/ATen/native/cpu/PowKernel.cpp

```diff
@@ -35,10 +35,8 @@ void pow_tensor_tensor_kernel(TensorIterator& iter) {
 }
 
 void pow_tensor_scalar_kernel(TensorIterator& iter, Scalar exp_scalar) {
-  // Casting exponent to double(not tensor.dtype) allows powering integral
-  // tensors to float exponent e.g. tensor([4]).pow(0.5) will be tensor([2])
-  const auto exp = exp_scalar.to<double>();
   if (isFloatingType(iter.dtype())) {
+    const auto exp = exp_scalar.to<double>();
```

Comment:
unused?

Reply:
It's used on lines 43, 50, 57 (which was what was done before this commit)